cypress/support/index.d.ts

```diff
@@ -3,7 +3,11 @@
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 declare namespace Cypress {
   interface Chainable {
-    loadSchedule(scheduleName?: import("./commands").ScheduleName): Chainable<Element>;
+    loadScheduleToMonth(
+      scheduleName?: import("./commands").ScheduleName,
```

Comment:
nie da się tego zaimportować jakoś na górze, a tutaj tylko użyć ScheduleName?

Reply:
Nawet nie wiesz ile z tym walczyłem XD To jest najładniejsze co udało mi się zrobić

Reply:
okej, to można zostawić